packages/components/file-uploader-input/src/file-uploader-input.js

```diff
@@ -210,30 +223,34 @@ customElementDefineHelper(
 		dragEnterHandler(e) {
 			if (!this.disabled && !this.disableDragAndDrop) {
 				e.preventDefault();
+				console.log('dragEnterHandler');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS2228)